### PR TITLE
Feature/swepub definitions

### PR DESCRIPTION
**Pull request by niklasl**
Let's open a PR and discuss some details here.

### CODE REVIEW

File: source/swepub/index.ttl [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+swepub:Published a swepub:PublicationStatus .
+swepub:Published a swepub:PublicationStatus .
+swepub:Preprint a swepub:PublicationStatus, swepub:OutputType .
+swepub:Submitted a swepub:PublicationStatus .
+swepub:Accepted a swepub:PublicationStatus .
+swepub:InPress a swepub:PublicationStatus .
+swepub:EpubAheadOfPrintOnlineFirst a swepub:PublicationStatus .
+
+swepub:ArtisticWork a swepub:PublicationType ;
+    :code "/kfu" .
```
Comment (niklasl) [inline source/swepub/index.ttl]: @klngwll Yes this looks like a typo.

**Comment by niklasl** [comment]
Given our intent in SwePub to use provisionActivity.date with string values (as in XL), along with the fact that date values in SwePub are typed (w3c dateTime or edtf), we most likely want to coerce date to AnyDateTime (or adjust KBV for just edtf:EDTF and skip the three different kbv types for EDTF levels 0, 1 and 2).

File: source/swepub/terms.ttl [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+</term/swepub/event> a :HostType ;
+    :label "event" .
+
+</term/swepub/dataset> a :HostType ;
+    :label "dataset" .
+
+
+</term/swepub/swedishlist/peer-reviewed> a :Classification ;
+    :prefLabel "Sakkunninggranskad publiceringskanal" .
+
```
Comment (klngwll) [inline source/swepub/terms.ttl]: Missing language tags.

File: source/generators.ttl [hunk 3]
```diff
@@ -9,3 +9,6 @@
 
 <https://id.kb.se/generator/globalchanges> a bf2:GenerationProcess ;
     rdfs:label "The Global Changes Process of Libris XL" .
+
+<https://id.kb.se/generator/swepub-classifier/1.0> a bf2:GenerationProcess ;
```
Comment (klngwll) [inline source/generators.ttl]: Maybe leave out versioning since it might not be used.

File: source/swepub/types.ttl [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+swepub:DoctoralThesis a :GenreForm ;
+    :exactMatch sppubl:dok, spout:publication\/doctoral-thesis .
+
+swepub:EditorialCollection a :GenreForm ;
+    :code "sam" ;
+    :exactMatch sppubl:sam, spout:publication\/edited-book .
+
+swepub:EditorialProceedings a :GenreForm ;
+    :code "pro" ;
```
Comment (sarwik) [inline source/swepub/types.ttl]: How come is code only defined for swepub:EditorialCollection and swepub:EditorialProceedings?

Comment (niklasl) [reply source/swepub/types.ttl]: Good call Actually, code should be removed here. The exactMatch links to the term with the code.

File: source/swepub/terms.ttl [hunk 5]
```diff
@@ -0,0 +1,44 @@
+prefix : <https://id.kb.se/vocab/>
+prefix swepub: <https://id.kb.se/term/swepub/>
+base <https://id.kb.se/>
+
+
+swepub:Published a swepub:PublicationStatus .
+
+swepub:Published a swepub:PublicationStatus .
```
[the remark this answers is not in the document]
Comment (niklasl) [reply source/swepub/terms.ttl]: Indeed. Cleaning up.

**Pull request merged**